src/GUI/GUIWindow.cpp

```diff
@@ -101,6 +101,146 @@ std::array<int, 28> possibleAddressingAwardBits = {{1,  2,  3,  4,  5,  7,  32,
                                                     38, 40, 41, 42, 43, 45, 46, 47, 48, 49,
                                                     50, 51, 52, 53, 54, 55, 56, 60}};
 
+IndexedArray<int, GUILD_FIRST, GUILD_LAST> priceForMembership = {{
+    {GUILD_OF_ELEMENTS, 100},
+    {GUILD_OF_SELF,     100},
+    {GUILD_OF_AIR,      50},
+    {GUILD_OF_EARTH,    50},
+    {GUILD_OF_FIRE,     50},
+    {GUILD_OF_WATER,    50},
+    {GUILD_OF_BODY,     50},
+    {GUILD_OF_MIND,     50},
+    {GUILD_OF_SPIRIT,   50},
+    {GUILD_OF_LIGHT,    1000},
+    {GUILD_OF_DARK,     1000}
+}};
+
+IndexedArray<int, CHARACTER_SKILL_FIRST, CHARACTER_SKILL_LAST> expertSkillMasteryCost = {{
```

Comment:
TODO - separate out arrays and _4B254D_SkillMasteryTeacher function?

Reply:
Ok, will add.
And that was a plan when will I get to simple houses refactoring.
Though I still musing whenever I should move it to UIHouses.cpp or somewhere else.